src/main/resources/add-ons-to-index.json

```diff
@@ -34,27 +38,35 @@
           "name": "Mark Goodrich"
         }
       ],
-      "backend": "org.openmrs.addonindex.backend.Modulus",
-      "modulusDetails": {
-        "id": "1"
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "openmrs",
+        "repo": "omod",
+        "package": "accesslogging"
       }
     },
     {
       "uid": "org.openmrs.module.addgutteritem",
+      "moduleId": "2",
```

Comment:
like my comment above, the moduleId should be `addgutteritem`. And same goes for all the rest.